src/collectors/windows.plugin/metadata.yaml

```diff
@@ -0,0 +1,265 @@
+plugin_name: windows.plugin
+modules:
+  - meta:
+      plugin_name: windows.plugin
+      module_name: PerflibProcesses
+      monitored_instance:
+        name: System statistics
+        link: "https://www.microsoft.com/en-us/windows"
```

Comment:
Isn't there a more specific link we can include?

Reply:
I updated the link. Thanks!